pkgs/LICENCE.md

```diff
@@ -0,0 +1,27 @@
+Copyright 2025, the Dart project authors.
```

Comment:
Can you move this to `pkgs/mcp_sdk`? I assume that's where you intended it to go.

Reply:
Yes thanks, done